src/basic/capability-util.c

```diff
@@ -335,7 +341,7 @@ int drop_privileges(uid_t uid, gid_t gid, uint64_t keep_capabilities) {
                     cap_set_flag(d, CAP_PERMITTED, j, bits, CAP_SET) < 0)
                         return log_error_errno(errno, "Failed to enable capabilities bits: %m");
 
-                if (cap_set_proc(d) < 0)
+                if (cap_set_proc(d) < 0 && (errno != EPERM || (detect_container() <= 0)))
```

Comment:
what is this about? why wouldn't we able to drop caps we have? (this is the same case as the same as the bounding set thing further up)